src/types.ts

```diff
@@ -1,11 +1,9 @@
-import Popper from 'popper.js';
+import * as Popper from '@popperjs/core';
 
-export type BasePlacement = 'top' | 'bottom' | 'left' | 'right';
+export type BasePlacement = Popper.Placement;
 
 export type Placement = Popper.Placement;
```

Comment:
This is now `Popper.VariationPlacement`, should it change here too?

It seems redundant to have `BasePlacement` and `Placement` equal each other.